crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -618,25 +627,39 @@ en:
           everything you need to use OpenStack APIs.
       backup:
         continue: 'Continue Upgrade Process'
-        header: 'Data Backup'
+        blockui: 'Continue Upgrade Process...'
+        header: 'Database Dump'
+        download: |
+          You can download the database dump from your node <b>%{node}</b>, you can find the file at
+          <b>%{path}</b>.
         description: |
           The SQL dump of the database was created. This will be reimported later when reapplying
           the OpenStack barclamps. It is strongly recommended to download that dump.
       nodes:
-        continue: 'Continue Upgrade Process'
-        header: 'Upgrading Nodes'
+        continue: 'Upgrade Nodes now'
+        blockui: 'Initializing node upgrade...'
+        header: 'Upgrading Nodes OS'
+        description: |
+          All service on the client nodes have been stopped now. As the next step the client nodes
+          will be upgraded to SLES 12 SP1. To start that process click 'Upgrade Nodes now' This
+          process will take a long time please be patient.
```

Comment:
Can we add some `,` here and there? I'd add one after `As the next step`, after `To start that process` and one before `please be patient`.

Also missing `.` after `'Upgrade Nodes now'`